src/app/components/top/solicitudes/solicitudes.component.ts

```diff
@@ -238,6 +238,13 @@ export class SolicitudesComponent implements OnInit {
         this.showIniciarPrestacion = false;
     }
 
+    // check if the type of prestarion is among the authorized prestations of the professional
+    isPresentationEnabled(prestacion) {
+        let enabled = false;
+        enabled = this.auth.check('rup:tipoPrestacion:' + prestacion.solicitud.tipoPrestacion.id);
+        return enabled;
```

Comment:
Esto puede quedar en una linea:

```suggestion
        return this.auth.check('rup:tipoPrestacion:' + prestacion.solicitud.tipoPrestacion.id);
```
o:
```suggestion
        const enabled = this.auth.check('rup:tipoPrestacion:' + prestacion.solicitud.tipoPrestacion.id);
        return enabled;
```
El this.auth.check retorna true/false siempre